unix/vncserver/selinux/vncsession.te

```diff
@@ -31,6 +31,7 @@ type vnc_home_t;
 userdom_user_home_content(vnc_home_t)
 
 allow vnc_session_t self:capability { chown dac_override dac_read_search fowner kill setgid setuid sys_resource };
+allow vnc_session_t self:dir { create write setattr };
```

Comment:
This means the service handles files in /proc manually?

Reply:
Is this about `/proc`? I assumed it just allows to create directories I specified in the other file.